integration-tests/tests/src/hooks/open-realm-before.ts

```diff
@@ -83,3 +84,34 @@ export function openRealmBefore(config: OpenRealmConfiguration = {}): void {
   before(openRealmBefore.name, openRealmHook(config));
   after("closeRealmAfter", closeThisRealm);
 }
+
+export async function setupRealmHook(this: AppContext & MultiRealmContext): Promise<void> {
+  this.openedInfo = [];
+
+  this.getRealm = async (config: OpenRealmConfiguration): Promise<Realm> => {
+    const user = await this.getUser(Realm.Credentials.anonymous(false));
+    const realmAndConfig = await openRealm(config, user);
+    this.openedInfo.push(realmAndConfig);
+    return realmAndConfig.realm;
+  };
+
+  this.closeAllRealms = async () => {
```

Comment:
`Realm.clearTestState()` and `Realm.shutdown()` are closing all open Realms.

Reply:
From what I can understand from the body of the function, then just calling `Realm.clearTestState` would be enough here, right?

If so, maybe it's also not necessary in the hook that's already there? https://github.com/realm/realm-js/blob/6269081d4e58c116ea7800bdd18c59dc0ffd9c80/integration-tests/tests/src/hooks/open-realm-before.ts#L31-L74